docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!

```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+==========================
```

Comment:
ReST formatting requires that the underlining is the same length as the text you are underlining - so you need a few more `=` here. 

If you want to check the formatting, you can run `make html` in the docs directory. You may need to `pip install sphinx` first.